src/Umbraco.Cms.ManagementApi/ViewModels/Dictionary/DictionaryViewModel.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.Models.Validation;
+using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.ManagementApi.ViewModels.Dictionary;
+
+/// <summary>
+///     The dictionary display model
+/// </summary>
+public class DictionaryViewModel : INotificationModel
+{
+    /// <summary>
+    ///     Initializes a new instance of the <see cref="DictionaryDisplay" /> class.
+    /// </summary>
+    public DictionaryViewModel()
+    {
+        Notifications = new List<BackOfficeNotification>();
+        Translations = new List<DictionaryTranslationViewModel>();
+        ContentApps = new List<ContentApp>();
+    }
+
+    /// <summary>
+    ///     Gets or sets the parent id.
+    /// </summary>
+    [JsonPropertyName("parentId")]
+    public Guid? ParentId { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the translations.
+    /// </summary>
+    [JsonPropertyName("translations")]
+    public IEnumerable<DictionaryTranslationViewModel> Translations { get; set; }
```

Comment:
```suggestion
    public IEnumerable<DictionaryTranslationViewModel> Translations { get; set; } = Enumerable.Empty<DictionaryTranslationViewModel>()
```

I think we should use this instead of newing up new lists in the constructor 😄